packages/fxa-auth-server/lib/payments/subscription-reminders.ts

```diff
@@ -48,7 +48,7 @@ export class SubscriptionReminders {
    * For all possible plan.intervals, determine if the plan is sufficiently
    * long based on planLength.
    */
-  private isEligiblePlan(plan: Plan): Boolean {
+  private isEligiblePlan(plan: Plan): boolean {
```

Comment:
Interesting. Probably well outside the scope of this PR, but worthy of a follow up ticket:
```sh
git grep -EIo ":\s?Boolean" packages | grep -E "\.[jt]sx?:" | sort | uniq

packages/browserid-verifier/lib/config.js:: Boolean
packages/fxa-admin-panel/server/config/index.ts:: Boolean
packages/fxa-admin-panel/src/components/PageAccountSearch/index.tsx:: Boolean
packages/fxa-admin-server/src/config/index.ts:: Boolean
packages/fxa-auth-server/config/index.ts:: Boolean
packages/fxa-content-server/server/lib/configuration.js:: Boolean
packages/fxa-customs-server/lib/config/config.js:: Boolean
packages/fxa-event-broker/src/config.ts:: Boolean
packages/fxa-graphql-api/src/config.ts:: Boolean
packages/fxa-payments-server/server/config/index.js:: Boolean
packages/fxa-profile-server/lib/config.js:: Boolean
packages/fxa-shared/db/config.ts:: Boolean
packages/fxa-shared/payments/iap/apple-app-store/app-store-helper.ts:: Boolean
packages/fxa-shared/tracing/config.ts:: Boolean
packages/fxa-support-panel/src/config.ts:: Boolean
```

Also, confirmed you covered the two cases of `Promise<Boolean>`:
```sh
git grep -EIo "Promise<Boolean>" packages | sort | uniq

packages/fxa-auth-server/lib/payments/stripe.ts:Promise<Boolean>
packages/fxa-auth-server/lib/payments/subscription-reminders.ts:Promise<Boolean>
```

Reply:
Correct, this ticket only covers the auth-server package. Other packages will need TS recommended linting enabled separately.

Reply:
Ah... OK, I was going to ask about [packages/fxa-auth-server/config/index.ts](https://github.com/mozilla/fxa/blob/FXA-6137/packages/fxa-auth-server/config/index.ts#L1545), but IIUC that looks like a Convict config file and `format: Boolean` is valid there (so my lazy grep would be invalid). :+1:

Reply:
Actually, this might be a better <kbd>grep</kbd>:

```sh
git grep -EIn ":\s?Boolean" packages/**/*.{js,jsx,ts,tsx} | grep -v "format: Boolean"

packages/fxa-admin-panel/src/components/PageAccountSearch/index.tsx:97:  query getAccountByEmail($email: String!, $autoCompleted: Boolean!) {
packages/fxa-shared/payments/iap/apple-app-store/app-store-helper.ts:16:      sandbox: Boolean;
```

Not sure about these which cast `as Boolean`:

```sh
git grep " as Boolean" packages | cat

packages/fxa-admin-server/src/gql/account/account.resolver.spec.ts:    const success = (await resolver.disableAccount(USER_1.uid)) as Boolean;
packages/fxa-admin-server/src/gql/account/account.resolver.spec.ts:    const success = (await resolver.enableAccount(USER_1.uid)) as Boolean;
```